tests/integration/conftest.py

```diff
@@ -39,7 +39,7 @@ async def credentials(ops_test: OpsTest):
 @pytest.fixture(scope="session")
 def cloud_configs_aws() -> tuple[dict[str, str], dict[str, str]]:
     configs = {
-        "endpoint": "https://s3.amazonaws.com",
+        "endpoint": os.getenv("AWS_ENDPOINT_URL", "https://s3.amazonaws.com"),
```

Comment:
I introduced this to use LocalStack, which claims to be AWS compatible. There's no equivalent on GCP as far as I understand, even though in theory all of these object storage solutions are more or less compatible. I can also remove it for consistency.